### DISCUSSION TITLE
Remove doctrine/dbal?

### DISCUSSION THREAD

**Discussion by hivokas** [laravel/nova-issues#6023] [Ideas & Feature Requests] [answered]
I was removing `doctrine/dbal` from my project, but noticed that it's also required by Nova. By quick search, I haven't found any usage of this package in Nova codebase (maybe I'm missing some indirect usage). 

Is it really needed by Nova, or it was needed previously and not needed anymore?

**Comment by niekdemelker** [comment]
I also checked this, `doctrine/dbal` doesn't appear to be used, and looking further `laravel/ui` also isn't used...

they both need to be removed from the composer.json if you ask me.

**Comment by schonhoff** [comment]
Laravel 11 removed `doctrine/dbal` but Laravel Nova didn't. Hopefully it will be removed some day...

**Comment by crynobone** [comment, marked as answer]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
